### DISCUSSION TITLE
Deploying the bolierplate to azure webapp

### DISCUSSION THREAD

**Discussion by mmohamedbasith** [vikejs/vike#1344] [Discussion]
@brillout  \ i am trying to deploy https://github.com/vikejs/vike/tree/main/boilerplates/boilerplate-react-ts to my azure webapp. However i am getting "You do not have permission to view this directory or page." error. 
Any suggestions? I was wondering how the Webapp will call the index file. I did see in server--> index file, 

From Dist client folder how the index file is served , do we need to any configuration?

**Comment by lourot** [comment]
Hey @mmohamedbasith I'm not familiar with Azure Web App. Are you trying to let it statically serve `dist/client/`? If yes, you need to add [pre-rendering](https://vike.dev/pre-rendering) to your Vike project, so that it will statically build the app into `dist/client/`.

Without pre-rendering, you need to tell Azure to run `npm run prod`.
